sign/rpmgensig.c

```diff
@@ -494,6 +505,36 @@ static rpmRC includeFileSignatures(Header *sigp, Header *hdrp)
 #endif
 }
 
+static rpmRC includeVeritySignatures(FD_t fd, Header *sigp, Header *hdrp)
+{
+#ifdef WITH_FSVERITY
+    rpmRC rc;
+    char *key = rpmExpand("%{?_file_signing_key}", NULL);
+    char *keypass = rpmExpand("%{?_file_signing_key_password}", NULL);
+    char *cert = rpmExpand("%{?_file_signing_cert}", NULL);
+
+    if (rstreq(keypass, "")) {
+	free(keypass);
+	keypass = NULL;
+    }
+
+    if (key && cert) {
```

Comment:
I'd go with just `if (key && *key && cert)` and remove the special casing of an empty string in the above.

Reply:
I am not sure I understand this comment. The above is keypass which is a pointer to the key password. It is only required if the key is password protected, so it's optional.

Reply:
Oh, sorry, my bad. Weary eyes at the end of the day getting things mixed up, do ignore.